apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3767,7 +3767,7 @@ int tc_filesystem_main(int argc, char *argv[])
 #ifdef CONFIG_ITC_FS
 	itc_fs_main();
 #endif
-#if defined(CONFIG_TC_FS_PROCFS) && !defined(CONFIG_SMARTFS_MULTI_ROOT_DIRS) && !defined(CONFIG_BUILD_PROTECTED)
+#if defined(CONFIG_TC_FS_PROCFS) && !defined(CONFIG_SMARTFS_MULTI_ROOT_DIRS) && !defined(CONFIG_BUILD_PROTECTED) && defined(CONFIG_FS_SMARTFS)
```

Comment:
When you define conditional operation like a if or something else, you should check cheaper first.
Actually CONFIG_SMARTFS_MULTI_ROOT_DIRS is tightly related to CONFIG_FS_SMARTFS

Reply:
Ok, I fixed it.
Thanks for comment.